lib/src/__tests__/DateTimePicker.test.tsx

```diff
@@ -83,7 +80,44 @@ describe('e2e - DateTimePicker', () => {
         .at(0)
         .text()
     ).toBe('2018');
-    // expect(component.find('ToolbarButton').at(1).text()).toBe('Jan 3');
+  });
+
+  it('prop: minDateTime – hours is disabled by date part', () => {
+    const { getByLabelText } = render(
+      <DesktopDateTimePicker
+        open
+        openTo="hours"
+        dateAdapter={new DateFnsUtils()}
+        renderInput={props => <TextField autoFocus {...props} />}
+        mask="__.__.____ __:__"
+        onChange={() => {}}
+        value={utilsToUse.date('2018-01-01T00:00:00.000Z')}
+        minDateTime={utilsToUse.date('2018-01-01T12:30:00.000Z')}
+      />,
+      { strict: false }
```

Comment:
Instead of setting strict here, defining it at the createClientRender level could reduce duplication and give a chance to document why it's required. For instance:

```jsx
// StrictModeViolation: uses Grow
```

https://github.com/mui-org/material-ui/blob/d34d0eef762705e9230ea218221c87f7d763a5fe/packages/material-ui/src/Snackbar/Snackbar.test.js#L11-L14